HW0/P6/P6B.py

```diff
@@ -16,16 +17,25 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    # Get 16 random wait times
+    wait_time = sorted([random.uniform(.00000000001, .001) for i in range(16)])
```

Comment:
This is an interesting choice for how to generate the wait times. Since we're plotting on a log scale, it would be nice to have logarithmically uniformly distributed values rather than linearly uniformly distributed values - I wonder whether the random module provides an analog to random.uniform that does that?